<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" rowspan="2">
     <widget class="QTextBrowser" name="mDumpBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="2">
     <widget class="Line" name="mBrowserSeparator">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="mPortGrpBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Port Filter</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="mPortLbl">
             <property name="minimumSize">
              <size>
               <width>99</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>99</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Port</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mPortEdit">
             <property name="minimumSize">
              <size>
               <width>219</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>219</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mAddrGrpBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Address Filter</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="mAddrLbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>99</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>99</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Address</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mAddrEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>219</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>219</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mPktGrpBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Packet Filters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="mObjIdxLbl">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>99</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>99</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Object Index</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mObjIdxEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>219</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>219</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="mSubIdxLbl">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>99</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>99</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Sub Index</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mSubIdxEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>219</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>219</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mTypeFiltGrpBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Type Filters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="mtSdoChkBox">
               <property name="text">
                <string>tSDO</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mtPdo1ChkBox">
               <property name="text">
                <string>tPDO1</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mtPdo2ChkBox">
               <property name="text">
                <string>tPDO2</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mtPdo3ChkBox">
               <property name="text">
                <string>tPDO3</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mtPdo4ChkBox">
               <property name="text">
                <string>tPDO4</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="mrSdoChkBox">
               <property name="text">
                <string>rSDO</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mrPdo1ChkBox">
               <property name="text">
                <string>rPDO1</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mrPdo2ChkBox">
               <property name="text">
                <string>rPDO2</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mrPdo3ChkBox">
               <property name="text">
                <string>rPDO3</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mrPdo4ChkBox">
               <property name="text">
                <string>rPDO4</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="mNmtChkBox">
               <property name="text">
                <string>NMT</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mEmerChkBox">
               <property name="text">
                <string>Emergency</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mTimeChkBox">
               <property name="text">
                <string>Time</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mNodeGuardChkBox">
               <property name="text">
                <string>Node Guard</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mTypeBtnGrp</string>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>48</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mActionOpen"/>
    <addaction name="mActionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="mStatusBar"/>
  <action name="mActionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="mActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="mSelectionBtnGrp"/>
  <buttongroup name="mTypeBtnGrp">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
